From c3441cb149f27d283d51ec4f6cecd6a5052de87a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sat, 21 Jan 2023 10:22:16 +0100
Subject: [PATCH] Fix dependency syntax error in test_dist_info

Fix the install_requires used in test_dist_info to use the dangling
syntax, in order to correctly handle markers.  This fixes syntax error
when parsed by packaging-22.0+, as well as setuptools warning:

```
UserWarning: One of the parsed requirements in `install_requires` looks like a valid environment marker: \'sys_platform != "linux"\'
Make sure that the config is correct and check https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#opt-2
  warnings.warn(msg, UserWarning)\n'
```
---
 setuptools/tests/test_dist_info.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setuptools/tests/test_dist_info.py b/setuptools/tests/test_dist_info.py
index 45b0d7fbb4..2ed790762f 100644
--- a/setuptools/tests/test_dist_info.py
+++ b/setuptools/tests/test_dist_info.py
@@ -142,7 +142,8 @@ class TestWheelCompatibility:
     version = {version}
 
     [options]
-    install_requires = foo>=12; sys_platform != "linux"
+    install_requires =
+        foo>=12; sys_platform != "linux"
 
     [options.extras_require]
     test = pytest
